packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -679,4 +679,42 @@ MeetingUtil.startInternalStats = (meeting) => {
   return meeting.internalStats;
 };
 
+MeetingUtil.handleAudioLogging = (audioTrack) => {
+  const LOG_HEADER = 'MeetingUtil#handleAudioLogging -->';
+
+  LoggerProxy.logger.log(LOG_HEADER);
+
+  if (audioTrack) {
+    const settings = audioTrack.getSettings();
+    const {deviceId} = settings;
+
+    LoggerProxy.logger.log(LOG_HEADER, deviceId);
```

Comment:
Consistency with other functions
```suggestion
    LoggerProxy.logger.log(LOG_HEADER, `deviceId = ${deviceId}`);
```

Reply:
Sounds good!